src/hssm/likelihoods/analytical.py

```diff
@@ -396,22 +396,21 @@ def logp_ddm_sdv(
 ddm_sdv_params = ddm_params + ["sv"]
 
 DDM: Type[pm.Distribution] = make_distribution(
-    "ddm",
-    logp_ddm,
+    rv="ddm",
```

Comment:
does `rv` always have to be passed now? Is there a way to intelligently figure this out from the model string?